contracts/test/utils.ts

```diff
@@ -27,3 +27,8 @@ export function expectEqualGrants(grant1: Grant, grant2: Grant) {
   expect(grant1.payee).to.equal(grant2.payee);
   expect(grant1.metaPtr).to.equal(grant2.metaPtr);
 }
+
+export async function timeTravel(provider: any, seconds: number) {
+  await provider.send('evm_increaseTime', [Number(seconds)]);
```

Comment:
Don't need to cast here since we specify the input as a number
```suggestion
  await provider.send('evm_increaseTime', [seconds]);
```